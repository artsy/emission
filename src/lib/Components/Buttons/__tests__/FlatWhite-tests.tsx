import "react-native"

import * as React from "react"
import * as renderer from "react-test-renderer"

import WhiteButton from "../FlatWhite"

it("renders properly", () => {
  const button = renderer.create(<WhiteButton selected={true} text={"I am a button"} />).toJSON()
  expect(button).toMatchSnapshot()
})
