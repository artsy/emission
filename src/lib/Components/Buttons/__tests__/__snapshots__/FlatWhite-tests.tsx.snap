// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(0, 0, 0, 1)",
        "borderColor": "#e5e5e5",
        "borderWidth": 1,
        "justifyContent": "center",
      },
    ]
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 12,
          },
          Object {
            "color": "black",
            "fontFamily": "AGaramondPro-Regular",
            "opacity": 1,
          },
          Object {
            "fontFamily": "Avant Garde Gothic ITCW01Dm",
          },
        ]
      }
    >
      I AM A BUTTON
    </Text>
  </View>
</View>
`;
